features/FEATURE_LWIP/lwip-interface/lwip/src/core/ipv6/lwip_nd6.c

```diff
@@ -1736,9 +1737,14 @@ nd6_new_router(const ip6_addr_t *router_addr, struct netif *netif)
     ip6_addr_set(&(neighbor_cache[neighbor_index].next_hop_address), router_addr);
     neighbor_cache[neighbor_index].netif = netif;
     neighbor_cache[neighbor_index].q = NULL;
-    neighbor_cache[neighbor_index].state = ND6_INCOMPLETE;
-    neighbor_cache[neighbor_index].counter.probes_sent = 1;
-    nd6_send_neighbor_cache_probe(&neighbor_cache[neighbor_index], ND6_SEND_FLAG_MULTICAST_DEST);
+    if (netif->hwaddr_len) {
```

Comment:
Suggested simple alternative - set state to STALE if interface has no HW address. I believe that should stay in STALE forever, without triggering probes. (Because the entries are not used, so don't transition to DELAY)